tests/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+"""
+Tests do not include two tests - test with
+float numbers as arguments (they can be unobviously
+generated by some other modules) and test with future
+date ('2056, 1, 1' for example). First test will always
+be failed, and second will always be passed. Also,
+since 'actual_password' is working with datetime.date.today()
+- it is necessary to update the test statements at least 1
+time per month, or rebuild the whole application.
+"""
+
+from app.main import actual_password
+
+
+def test_60_days_or_more():
+    assert actual_password(2021, 9, 5) == "Immediately change the password!"
```

Comment:
Tests should not depend on the current date if it is possible. Use `date.today()` and `timedelta` to make tests correct